### ISSUE TITLE
Customize printing experience

### ISSUE THREAD

**Issue by JeremyGenius** [JeremyGenius/ate#19]
Right now, the only way you can print a document is to select the option from your browser menu (if the browser even has this). The page that is then printed shows up with the gray color scheme, the monospace font, and the app logo. Instead, there should be a print button in the editor menu, and customized print styling. It should probably just be a white page with a sans-serif font and nothing else.

**Comment by JeremyGenius** [comment]
Implemented!
